src/spdigger_stack.c

```diff
@@ -2187,9 +2186,24 @@ struct Thing* check_place_to_pickup_gold(struct Thing* thing, MapSubtlCoord stl_
     return INVALID_THING;
 }
 
-struct Thing *check_place_to_pickup_spell(struct Thing *thing, long a2, long a3)
+struct Thing *check_place_to_pickup_spell(struct Thing *creatng, MapSubtlCoord stl_x, MapSubtlCoord stl_y)
 {
-    return _DK_check_place_to_pickup_spell(thing, a2, a3);
+    // return _DK_check_place_to_pickup_spell(thing, stl_x, stl_y);
```

Comment:
since the DLLIMPORT struct Thing *_DK_check_place_to_pickup_spell(struct Thing *creatng, long slb_x, long slb_y); is gone anyways it's not like uncommenting that code would work, if it was ever needed to go back the git history would be used, so line is useless imo